lib/parameter_substitution/formatters/if_truthy.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class ParameterSubstitution::Formatters::IfTruthy < ParameterSubstitution::Formatters::Base
```

Comment:
I'm on the fence about the name. I don't believe that technically this respects "truthiness" in a pure Ruby way. But, naming it "if_true" seemed too narrowly scoped, as I think we should support non-strictly-boolean values like "Y" and 
"yes".